README.md

```diff
@@ -56,7 +56,7 @@ const Card = (props) => {
   return (
     <div className={styles.cardContainer}>
       {cardDetails && cardDetails.map((card, index) =>
-        <div className={styles.card} key={`key-${index}`}>
+        <div className={styles.card} key={styles.id}>
```

Comment:
Is it `styles.id` or `card.id` ? since styles are just an object which holds the style property definitions right.